### ISSUE TITLE
How to use environment vars from context?

### ISSUE THREAD

**Issue by bryanlatten** [honojs/node-server#72]
Hello! Thank you for this excellent tool - it's really helping solve a particular need.

Curious if there was a way to pipe `process.env` into the "regular" context object? It's just always undefined currently.

Thank you!

**Comment by yusukebe** [comment]
Hi @bryanlatten 

If you're running it on Node.js, you'll need to retrieve it from `process.env`. In this context, `c.env` will always be `undefined` as it is specific to Cloudflare Workers. However, you can use the [`env` provided by the adapter](https://hono.dev/api/adapter#env). The purpose of this is to provide a unified way to access environment variables across different runtimes.

```ts
import { env } from 'hono/adapter'

app.get('/env', (c) => {
  const { NAME } = env<{ NAME: string }>(c)
  return c.text(NAME)
})
```

**Comment by bryanlatten** [comment]
@yusukebe appreciate the fast response. we'll take it!

**Comment by muhaimincs** [comment]
> Hi @bryanlatten
> 
> If you're running it on Node.js, you'll need to retrieve it from `process.env`. In this context, `c.env` will always be `undefined` as it is specific to Cloudflare Workers. However, you can use the [`env` provided by the adapter](https://hono.dev/api/adapter#env). The purpose of this is to provide a unified way to access environment variables across different runtimes.
> 
> ```ts
> import { env } from 'hono/adapter'
> 
> app.get('/env', (c) => {
>   const { NAME } = env<{ NAME: string }>(c)
>   return c.text(NAME)
> })
> ```

To me it's still undefined. How do I begin to use it?

**Comment by yusukebe** [comment]
@bryanlatten 

It does not depend on whether you use middleware or not.

**Comment by bryanlatten** [comment]
@yusukebe just to be clear, my issue was solved. @muhaimincs is the new reporter, might be a different issue

**Comment by yusukebe** [comment]
@bryanlatten 

Ah, I've missed whom I should mention. Sorry!

**Comment by yusukebe** [comment]
@muhaimincs 

It does not depend on whether you use middleware or not. If it's still not fixed, let me know the platform/runtime you run the app.

**Comment by muhaimincs** [comment]
Yeah it's still undefined. I thought this library provide the built-in .env file to be loaded while it run. How to properly load the .env file?
